Users/Tutorial.rst

```diff
@@ -54,14 +54,14 @@ There are two kinds of bears:
 - LocalBears, which only perform analysis on each file itself
 - GlobalBears, which are project wide, like the GitCommitBear
 
-Let's find out what bears are available for C:
+Let's find out what bears are available for C and Python:
 
 ::
 
-    coala --show-bears --filter-by-language C
+    coala --show-bears --filter-by-language C Python
 
-coala now lists all bears that can operate on C code. You can specify the
-kind of analysis with the ``--bears`` argument:
+coala now lists all bears that can operate on C and Python code. You can
```

Comment:
It may be "or" as command is listing all bears taking union of C and python language. I got little bit confused about that.